ci/gh_prepare_android_artifacts.sh

```diff
@@ -43,33 +43,15 @@ function prepare_capture_sdk() {
 function prepare_capture_timber() {
   echo "+++ Preparing Android Capture Timber library artifacts for '$version' version"
 
-  pushd "$(mktemp -d)"
-    local -r out_artifacts_dir="capture-timber-out"
-
-    unzip "$sdk_repo/capture-timber.zip"
-
-    mkdir "$out_artifacts_dir"
-
-    local -r name="capture-timber-$version"
-
-    files=(\
-      "$name.aar" \
-      "$name.module" \
-      "$name.pom" \
-      "$name-javadoc.jar" \
-      "$name-sources.jar" \
-      "$sdk_repo/ci/LICENSE.txt" \
-      "$sdk_repo/ci/NOTICE.txt" \
-    )
+  cp "$sdk_repo/capture-timber.zip" "$sdk_repo/capture-timber-$version.android.zip"
```

Comment:
This has the implication that we will now include checksum files in the GH release zips for timber/plugin which isn't ideal but I don't think anyone relies on the GH releases so seems fine until we can simplify the deployment pipeline further

Reply:
that should be fine, arguably any artifact bundle can benefit from including checksums